crates/iota-sdk-ffi/src/types/type_tag.rs

```diff
@@ -3,6 +3,8 @@
 
 use std::sync::Arc;
 
+use iota_types::StructTag;
```

Comment:
What's this for?

Reply:
My IDE probably added it, not sure why rustc wasn't complaining about it not being used tbh
https://github.com/iotaledger/iota-rust-sdk/pull/83/commits/d2f3a8bfff68f682cafb66ab58896a5de493165f